mailet/api/pom.xml

```diff
@@ -55,11 +55,16 @@
             <version>1.7.5</version>
```

Comment:
ok, that's what you did, so you just need to add every cache invalidation that is missing. And check there is no concurrency issue...